test/Sentry.Testing/FakeFileSystem.cs

```diff
@@ -1,54 +1,95 @@
-using MockFileSystem = System.IO.Abstractions.TestingHelpers.MockFileSystem;
+using System.IO.Abstractions.TestingHelpers;
 
-namespace Sentry.Testing;
+namespace Sentry.Internal;
```

Comment:
Shouldn't the namespace match the assembly and folder? If I see a class in the Sentry.Internal namespace, my assumption is that it lives in the Sentry package...